models/iris_with_gimbal/model.sdf

```diff
@@ -215,14 +215,15 @@
         <offset>0</offset>
```

Comment:
I'd prefer to see the changes to the models split into a separate PR. That way we can proceed with  including the motor plugin without altering the existing models and their behaviour. When we're more comfortable with the plugin we can consider migrating the examples over to it.

In the meanwhile a small test model (one of the iris motors and prop fixed to a thrust stand would be good). That would allow testing and demonstrations. It could even have a force torque sensor plugin included in the joint for additional feedback.